components/support/migration/metrics.yaml

```diff
@@ -0,0 +1,508 @@
+      - fenix-core@mozilla.com
+    expires: never
+    lifetime: ping
+  duration:
+    type: timespan
+    time_unit: millisecond
+    description: >
+      Duration of the bookmarks migration.
+    send_in_pings:
+      - migration
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/5888
+    data_reviews:
+      - https://github.com/mozilla-mobile/android-components/pull/5483
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: never
+    lifetime: ping
+  any_failures:
+    type: boolean
+    description: >
+      Did this migration encounter any failures (exceptions)?
+    send_in_pings:
+      - migration
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/5888
+    data_reviews:
+      - https://github.com/mozilla-mobile/android-components/pull/5483
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: never
+    lifetime: ping
+
+migration.open_tabs:
+  detected:
+    type: counter
+    description: >
+      Number of detected open tabs.
+    send_in_pings:
+      - migration
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/5888
+    data_reviews:
+      - https://github.com/mozilla-mobile/android-components/pull/5483
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: never
+    lifetime: ping
+  migrated:
+    type: counter
+    description: >
+      Number of migrated open tabs.
+    send_in_pings:
+      - migration
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/5888
+    data_reviews:
+      - https://github.com/mozilla-mobile/android-components/pull/5483
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: never
+    lifetime: ping
+  any_failures:
+    type: boolean
+    description: >
+      Did this migration encounter any failures (exceptions)?
+    send_in_pings:
+      - migration
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/5888
+    data_reviews:
+      - https://github.com/mozilla-mobile/android-components/pull/5483
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: never
+    lifetime: ping
+
+migration.fxa:
+  any_failures:
+    type: boolean
+    description: >
+      Did this migration encounter any failures (exceptions)?
+    send_in_pings:
+      - migration
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/5888
+    data_reviews:
+      - https://github.com/mozilla-mobile/android-components/pull/5483
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: never
+    lifetime: ping
+  failure_reason:
+    type: string
+    description: >
+      Why this migration failed.
+    send_in_pings:
+      - migration
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/5888
+    data_reviews:
+      - https://github.com/mozilla-mobile/android-components/pull/5483
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: never
+    lifetime: ping
+  success_reason:
```

Comment:
Is there a pre-defined list of reasons why this could succeed? Would it make sense to send success codes instead?